src/Helper/FileDescriptorReader.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Helper;
+
+/**
+ * Class FileDescriptorReader.
+ */
+class FileDescriptorReader
+{
+    /**
+     * @param string $inputFd
+     *
+     * @return bool|string
+     */
+    private static function readFileDescriptor(string $inputFd)
+    {
+        $fileHandler = fopen($inputFd, 'r');
+
+        return stream_get_contents($fileHandler);
+    }
+
+    /**
+     * @return bool|string
+     */
+    public function readFd3()
```

Comment:
The methods in this class could be static

Reply:
Mh, but apparently static methods cannot be mocked in unit tests.